requirements.txt

```diff
@@ -1,5 +1,6 @@
 pandas
 six
+retry
```

Comment:
Looks like this hasn't been updated in 3 years. Is this something that we really need as a dependency? If so, is there a better maintained alternative.

Reply:
I could change it over to tenacity. I think I left this here by accident while I had GCS code in my dev, so might not need it in this repo at all.

Reply:
Cool. Let's double check and remove if possible.